docs/sources/enterprise/reporting.md

```diff
@@ -77,6 +77,15 @@ Weekly reports are generated once per week. All fields are required.
 
 Reports which are scheduled to never be sent have no parameter and will not be sent to the scheduler. They may be manually generated from the **Send test email** prompt or via the API.
 
+### Layout
+
+> We're actively working on developing new report layout options. [Contact us](https://grafana.com/contact?about=grafana-enterprise) if you would like to get involved in the design process.
```

Comment:
Should they use keywords or something to help route the message properly?

Reply:
Good point, I'm working on getting that working for us

Reply:
✅ I've added the parameters to the contact form now, so it should work when both grafana.com and the docs are running with my changes (and it'll work but not with our custom pre-formatted text until then).